## **Aqui est√° uma sele√ß√£o de 12 artigos essenciais que cobrem t√≥picos importantes na √°rea de melhoramento gen√©tico.**

## üå± Sele√ß√£o Gen√¥mica & Gen√©tica Quantitativa

# ‚ù∂ Entendendo o Processo de Sele√ß√£o Gen√¥mica em Plantas  
- https://www.embrapa.br/busca-de-publicacoes/-/publicacao/1155985/melhoramento-de-precisao-aplicacoes-e-perspectivas-na-genetica-de-plantas
- Resende, R.T. Balancing genomic selection efforts for allogamous plant breeding programs. J. Crop Sci. Biotechnol. 27, 349‚Äì358 (2024). https://doi.org/10.1007/s12892-024-00234-7

# ‚ù∑ Como Funciona a Gen√©tica Quantitativa?  
- https://www.linkedin.com/posts/thecrop_breeding-precisionbreeding-agriculture-activity-7125966987658178563-_H7R?utm_source=share&utm_medium=member_desktop

# ‚ù∏ Deriva√ß√µes das Rela√ß√µes Gen√©ticas Entre Indiv√≠duos  
- https://www.linkedin.com/posts/thecrop_breeding-precisionagriculture-genomics-activity-7128515032455245825-8aaz?utm_source=share&utm_medium=member_desktop

## üß¨ Modelos Estat√≠sticos & Teoria

# ‚ùπ Efeitos Fixos vs Aleat√≥rios em Modelos Mistos  
- https://www.linkedin.com/posts/thecrop_mixedmodels-breeding-precisionbreeding-activity-7134863118593081345-osE1?utm_source=share&utm_medium=member_desktop

# ‚ù∫ A Equa√ß√£o do Melhorista Explicada  
- https://www.linkedin.com/posts/thecrop_breeding-phenomics-genomics-activity-7158601205236056064-TzZM?utm_source=share&utm_medium=member_desktop

# ‚ùª Por Que a Herdabilidade √© Fundamental no Melhoramento?  
- https://www.linkedin.com/posts/thecrop_genetics-breeding-heritability-activity-7152750689352327168-m7ld?utm_source=share&utm_medium=member_desktop

## üåæ Popula√ß√µes de Melhoramento & M√©todos

# ‚ùº Entendendo Popula√ß√µes de Melhoramento  
- https://www.linkedin.com/posts/thecrop_thecrop-plantbreeding-genetics-activity-7155311141504577536-p5YJ?utm_source=share&utm_medium=member_desktop

# ‚ùΩ A Complexa Intera√ß√£o Gen√≥tipo √ó Ambiente  
- https://www.linkedin.com/posts/thecrop_gxe-breeding-plantbreeding-activity-7175942157944446976-B_z5?utm_source=share&utm_medium=member_desktop

# ‚ùæ Sele√ß√£o Recorrente ‚Äî √â T√£o Simples Assim?  
- https://www.linkedin.com/posts/thecrop_breeding-genomics-phenomics-activity-7188680080171163648-Cblr?utm_source=share&utm_medium=member_desktop

## üìä ‚Äò√îmicas & Implementa√ß√£o

# ‚ù∂‚ìø Fen√¥mica no Melhoramento Gen√©tico  
- https://www.linkedin.com/posts/thecrop_phenomics-breeding-animalbreeding-activity-7198382693988888576-o0Pu?utm_source=share&utm_medium=member_desktop

# ‚ù∂‚ù∂ A Ascens√£o da Enviromia (Enviromics)  
- https://www.cell.com/molecular-plant/fulltext/S1674-2052(24)00118-7?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS1674205224001187%3Fshowall%3Dtrue

# ‚ù∂‚ù∑ Etapas de Implementa√ß√£o Experimental para o Melhoramento de Culturas  
- https://nph.onlinelibrary.wiley.com/doi/10.1111/nph.19951

## **Artigos de pesquisa e de revis√£o voltados √† gen√©tica e melhoramento, gen√©tica quantitativa e gen√¥mica de diversas culturas.**

# Feij√£o (*Phaseolus vulgaris*)

Adam-Blondon, A. F., S√©vignac, M., Thoquet, P., Poncet, V., & Geffroy, V. (1994). SCAR, RAPD and RFLP markers linked to a dominant gene (Are) conferring resistance to anthracnose in common bean. Theoretical and Applied Genetics, 88(6‚Äì7), 865‚Äì870. https://doi.org/10.1007/BF01253998

Banoo, A., Sharma, S., Kaul, S., Khajuria, P., & Gupta, P. (2020). North-Western Himalayan common beans: Population structure and mapping of quantitative anthracnose resistance through genome wide association study. Frontiers in Plant Science, 11, 571618. https://doi.org/10.3389/fpls.2020.571618

Bansuli, & Kapila, R. K. (2024). Marker assisted transfer of anthracnose resistance gene Co-5 to elite French bean (Phaseolus vulgaris L.) background. Euphytica, 220, 71. https://doi.org/10.1007/s10681-024-03321-6

Bello, M. H., Moghaddam, S. M., Massoudi, M., McClean, P. E., Cregan, P. B., & Miklas, P. N. (2014). Application of in silico bulked segregant analysis for rapid development of markers linked to Bean common mosaic virus resistance in common bean. BMC Genomics, 15, 903. https://doi.org/10.1186/1471-2164-15-903

Bitocchi, E., Nanni, L., Bellucci, E., Rossi, M., Giardini, A., Zeuli, P. S., ... & Papa, R. (2012). Mesoamerican origin of the common bean (Phaseolus vulgaris L.) is revealed by sequence data. Proceedings of the National Academy of Sciences, 109(14), E788‚ÄìE796. https://doi.org/10.1073/pnas.1108973109

Campa, A., Giraldez, R., & Ferreira, J. J. (2009). Genetic dissection of the resistance to nine anthracnose races in the common bean differential cultivars MDRK and TU. Theoretical and Applied Genetics, 119, 1‚Äì11. https://doi.org/10.1007/s00122-009-1011-8

Castro, S. A. L., Ramalho, M. A. P., Abreu, A. F. B., & Moreira, M. A. (2017). Genetics and mapping of a new anthracnose resistance locus in Andean common bean Paloma. BMC Genomics, 18, 306. https://doi.org/10.1186/s12864-017-3685-7

David, P., Rech, P., Naoumkina, M., Buddenhagen, C., & Geffroy, V. (2009). A nomadic subtelomeric disease resistance gene cluster in common bean. Plant Physiology, 151, 1048‚Äì1065. https://doi.org/10.1104/pp.109.142109

De Jesus Junior, W. C., Sartorato, A., & Alzate-Marin, A. L. (2001). Effects of angular leaf spot and rust on yield loss of Phaseolus vulgaris. Phytopathology, 91(11), 1045‚Äì1053. https://doi.org/10.1094/PHYTO.2001.91.11.1045

De Souza, I. P., Rodrigues, M. C., Pereira, H. S., Brum, C. B., & Melo, L. C. (2023). Whole-genome resequencing of common bean elite breeding lines. Scientific Reports, 13, 12721. https://doi.org/10.1038/s41598-023-39399-6

De Souza, T. L. P. O., Damasceno-Silva, K. J., Abreu, A. F. B., & Moreira, M. A. (2011). Characterization of the rust resistance gene present in the common bean cultivar Ouro Negro, the main rust resistance source used in Brazil. Plant Pathology, 60(5), 839‚Äì845. https://doi.org/10.1111/j.1365-3059.2011.02456.x

Erfatpour, M., Simons, K. J., Roy, J., Figueroa Cerna, J. C., Lee, R., Beaver, J., ... & Osorno, J. M. (2025). Fine mapping and identification of causal alleles at the Ur-11 locus controlling rust resistance in common bean (Phaseolus vulgaris L.). Theoretical and Applied Genetics, 138, 55. https://doi.org/10.1007/s00122-025-04836-9

Fritsche-Neto, R., Souza, T. L. P. O., Pereira, H. S., Jannink, J. L., & Melo, L. C. (2019). Association mapping in common bean revealed regions associated with resistance to anthracnose and angular leaf spot. Scientia Agricola, 76(4), 321‚Äì327. https://doi.org/10.1590/1678-992X-2017-0306

Geffroy, V., et al. (1999). Identification of an ancestral resistance gene cluster involved in the coevolution process between Phaseolus vulgaris and its fungal pathogen Colletotrichum lindemuthianum. Molecular Plant-Microbe Interactions, 12(9), 774‚Äì784. https://doi.org/10.1094/MPMI.1999.12.9.774

Geffroy, V., et al. (2008). Resistance to Colletotrichum lindemuthianum in Phaseolus vulgaris: a case study for mapping two independent genes. Theoretical and Applied Genetics, 116, 407‚Äì415. https://doi.org/10.1007/s00122-007-0678-y

Geffroy, V., et al. (2009). Molecular analysis of a large subtelomeric Nucleotide-Binding-Site‚ÄìLeucine-Rich-Repeat family in two representative genotypes of the major gene pools of Phaseolus vulgaris. Genetics, 181, 405‚Äì419. https://doi.org/10.1534/genetics.108.093583

Geffroy, V., et al. (2000). Inheritance of partial resistance against Colletotrichum lindemuthianum in Phaseolus vulgaris and co-localization of QTL with genes involved in specific resistance. Molecular Plant-Microbe Interactions, 13(3), 287‚Äì296. https://doi.org/10.1094/MPMI.2000.13.3.287

Gilio, T. A. S., et al. (2020). Fine mapping of an anthracnose-resistance locus in Andean common bean cultivar Amendoim Cavalo. PLoS ONE, 15(10), e0239763. https://doi.org/10.1371/journal.pone.0239763

Gon√ßalves-Vidigal, M. C., et al. (2011). Linkage mapping of the Phg-1 and Co-14 genes for resistance to angular leaf spot and anthracnose in the common bean cultivar AND 277. Theoretical and Applied Genetics, 122, 893‚Äì903. https://doi.org/10.1007/s00122-010-1496-1

Gon√ßalves-Vidigal, M. C., et al. (2013). Co-segregation analysis and mapping of the anthracnose Co-10 and angular leaf spot Phg-ON disease-resistance genes in the common bean cultivar Ouro Negro. Theoretical and Applied Genetics, 126, 2245‚Äì2255. https://doi.org/10.1007/s00122-013-2131-8

Gupta, C., Salgotra, R. K., & Venegas, R. A. (2025). SNP-based functional marker for anthracnose-resistant Co-2 gene in common bean (Phaseolus vulgaris L.). Journal of Plant Diseases and Protection, 132, 66. https://doi.org/10.1007/s41348-025-01060-x

Hart, J. P., & Griffiths, P. D. (2015). Genotyping-by-sequencing enabled mapping and marker development for the By-2 potyvirus resistance allele in common bean. The Plant Genome, 8(1). https://doi.org/10.3835/plantgenome2014.09.0058

Iwata, A., et al. (2013). Identification and characterization of functional centromeres of the common bean. The Plant Journal, 76, 47‚Äì60. https://doi.org/10.1111/tpj.12269

Jeffery, H. R., et al. (2025). A comprehensive review of the genetic control of cooking time in dry beans (Phaseolus vulgaris L.). Plants, People, Planet. https://doi.org/10.1002/ppp3.10631

K√§llman, T., et al. (2013). A significant fraction of 21-nucleotide small RNA originates from phased degradation of resistance genes in several perennial species. Plant Physiology, 162(2), 741‚Äì754. https://doi.org/10.1104/pp.113.214643

Kuwabo, K., et al. (2023). Genome-wide association analysis of anthracnose resistance in the Yellow Bean Collection of Common Bean. PLoS ONE, 18(11), e0293291. https://doi.org/10.1371/journal.pone.0293291

Lacanallo, G. F., & Gon√ßalves-Vidigal, M. C. (2015). Mapping of an Andean gene for anthracnose resistance (Co-13) in common bean (Phaseolus vulgaris L.) Jalo Listras Pretas landrace. Australian Journal of Crop Science, 9(5), 394‚Äì400.

Lateef, I., et al. (2024). Fine mapping of a new common bean anthracnose resistance gene (Co-18) to the proximal end of Pv10 in Indian landrace KRC 5. Theoretical and Applied Genetics, 137, 32. https://doi.org/10.1007/s00122-023-04539-z

Leal, A. A., Mangolin, C. A., Gon√ßalves, L. S. A., & Scapim, C. A. (2020). Resistance loci to anthracnose and angular leaf spot in Brazilian common bean cultivar BRSMG Realce. Agronomy, 10(3), 370. https://doi.org/10.3390/agronomy10030370

Mao, Y., Zhang, H., Xu, N., Xu, L. A., Wu, J., Yan, L., & Tian, F. (2019). PvMATE8 is required for anthracnose resistance and iron homeostasis in Phaseolus vulgaris. Plant and Cell Physiology, 60(9), 1941‚Äì1954. https://doi.org/10.1093/pcp/pcz106

Melotto, M., Coelho, M. F. B., Pedrosa-Harand, A., Kelly, J. D., & Camargo, L. E. A. (2004). Fine linkage mapping of the anthracnose resistance gene Co-4 in common bean (Phaseolus vulgaris L.) using a BAC library. Theoretical and Applied Genetics, 108, 817‚Äì824. https://doi.org/10.1007/s00122-003-1485-2

Melotto, M., Afanador, L., & Kelly, J. D. (1996). Development of a SCAR marker linked to the I gene in common bean. Genome, 39(6), 1216‚Äì1219. https://doi.org/10.1139/g96-152

Melotto, M., & Kelly, J. D. (2001). Fine mapping of the Co-4 locus for anthracnose resistance in common bean. Theoretical and Applied Genetics, 103, 800‚Äì807. https://doi.org/10.1007/s001220100615

Mend√©z-Vigo, B., Soengas, P., & Santalla, M. (2005). Identification, characterization and mapping of genes conferring resistance to anthracnose in a Spanish landrace of common bean. Plant Breeding, 124(6), 556‚Äì561. https://doi.org/10.1111/j.1439-0523.2005.01140.x

Miklas, P. N., Stone, V., Daly, M., Wang, R. Q., & Johnson, E. (2000). QTL conditioning physiological resistance and avoidance to white mold in common bean. Crop Science, 40(2), 539‚Äì542. https://doi.org/10.2135/cropsci2000.402539x

Nunes, M. P., et al. (2007). Genetic analysis of angular leaf spot resistance in common bean (Phaseolus vulgaris L.). Euphytica, 157, 47‚Äì56. https://doi.org/10.1007/s10681-007-9395-0

Pastor-Corrales, M. A., & Tu, J. C. (1989). Anthracnose. In H. F. Schwartz & M. A. Pastor-Corrales (Eds.), Bean production problems in the tropics (pp. 77‚Äì104). CIAT.

Raman, H., Raman, R., Kilian, A., Detering, F., & Balint-Kurti, P. (2014). Genome-wide association mapping identifies loci for resistance to rust, anthracnose and angular leaf spot in common bean. Theoretical and Applied Genetics, 127(5), 1005‚Äì1019. https://doi.org/10.1007/s00122-014-2262-5

Richardson, A. O., & Palmer, J. D. (2007). Horizontal gene transfer in plants. Journal of Experimental Botany, 58(1), 1‚Äì9. https://doi.org/10.1093/jxb/erl148

Rodrigues, M. C., Gon√ßalves-Vidigal, M. C., & Pereira, H. S. (2022). High-throughput genotyping to identify resistance loci to angular leaf spot in common bean. Theoretical and Applied Genetics, 135, 1525‚Äì1537. https://doi.org/10.1007/s00122-022-04022-7

Rudolph, L. K., Kelly, J. D., & Miklas, P. N. (2022). Understanding and exploiting race-specific and non-race-specific resistance for control of anthracnose and angular leaf spot of common bean. Frontiers in Plant Science, 13, 823918. https://doi.org/10.3389/fpls.2022.823918

Souza, T. L. P. O., et al. (2014). Mapping QTLs for resistance to anthracnose and angular leaf spot in common bean (Phaseolus vulgaris L.). Euphytica, 198(3), 385‚Äì396. https://doi.org/10.1007/s10681-014-1117-y

Souza, T. L. P. O., et al. (2023). Identification of new sources of resistance to angular leaf spot and powdery mildew in common bean. Plant Pathology, 72(4), 706‚Äì714. https://doi.org/10.1111/ppa.13652

Tock, A. J., Fourie, D., Walley, P. G., et al. (2017). Genome-wide association mapping reveals genes underlying anthracnose resistance in Andean beans. Scientific Reports, 7, 45718. https://doi.org/10.1038/srep45718

Tock, A. J., et al. (2021). A bean pan-genome provides insight into common bean diversity. Nature Genetics, 53, 1494‚Äì1505. https://doi.org/10.1038/s41588-021-00947-7

Valentini, G., et al. (2019). Fine mapping of the resistance gene Co-3X conferring resistance to anthracnose in common bean. Molecular Breeding, 39, 78. https://doi.org/10.1007/s11032-019-0995-4

Vasconcelos, J. R. B., et al. (2017). QTL mapping of angular leaf spot resistance in common bean under different environments. Crop Science, 57(4), 1932‚Äì1944. https://doi.org/10.2135/cropsci2016.10.0862

Zhang, P., et al. (2023). Genetic basis of multiple disease resistance in common bean revealed through genome-wide association analysis and genomic prediction. Frontiers in Plant Science, 14, 1173035. https://doi.org/10.3389/fpls.2023.1173035

# Milho (*Zea mays*)
1. Moeinizade, S., Kusmec, A., Hu, G., Wang, L., & Schnable, P. S. (2020). [Multi-trait genomic selection methods for crop improvement](https://academic.oup.com/genetics/article/215/4/931/6065544). Genetics.

2. Farooq, M., van Dijk, A. D., Nijveen, H., Mansoor, S., & de Ridder, D. (2023). [Genomic prediction in plants: opportunities for ensemble machine learning based approaches](https://f1000research.com/articles/11-802). F1000Research.

3. Lane, H. M., Murray, S. C., Montesinos‚ÄëL√≥pez, O. A., Montesinos‚ÄëL√≥pez, A., Crossa, J., Rooney, D. K., ... & Morgan, C. L. (2020). [Phenomic selection and prediction of maize grain yield from near‚Äêinfrared reflectance spectroscopy of kernels](https://acsess.onlinelibrary.wiley.com/doi/10.1002/ppj2.20002). The Plant Phenome Journal.

4. Wei√ü, T. M., Zhu, X., Leiser, W. L., Li, D., Liu, W., Schipprack, W., ... & W√ºrschum, T. (2022). [Unraveling the potential of phenomic selection within and among diverse breeding material of maize (Zea mays L.)](https://academic.oup.com/g3journal/article/12/3/jkab445/6509517). G3.

5. Sun, H., Wei, M., Xu, Z., Bai, C., & Sun, B. (2022). [PC‚ÄêDOT: Improving genomic prediction ability of principal component regression by DOT product](https://onlinelibrary.wiley.com/doi/10.1111/age.13255). Animal Genetics.

6. Cheng, J., Maltecca, C., VanRaden, P. M., O'Connell, J. R., Ma, L., & Jiang, J. (2023). [SLEMM: million-scale genomic predictions with window-based SNP weighting](https://academic.oup.com/bioinformatics/article/39/3/btad127/7075542). Bioinformatics.

7. Robert, P., Brault, C., Rincent, R., & Segura, V. (2022). [Phenomic Selection: A New and Efficient Alternative to Genomic Selection](https://link.springer.com/protocol/10.1007/978-1-0716-2205-6_14). In Genomic Prediction of Complex Traits: Methods and Protocols (pp. 397-420). New York, NY: Springer US.

8. Westhues, C. C., Mahone, G. S., da Silva, S., Thorwarth, P., Schmidt, M., Richter, J. C., ... & Beissinger, T. M. (2021). [Prediction of maize phenotypic traits with genomic and environmental predictors using gradient boosting frameworks](https://www.frontiersin.org/articles/10.3389/fpls.2021.699589/full). Frontiers in Plant Science.

9. Piepho, H. P., B√ºchse, A., & Emrich, K. (2003). [A hitchhiker's guide to mixed models for randomized experiments](https://onlinelibrary.wiley.com/doi/abs/10.1046/j.1439-037X.2003.00049.x). Journal of Agronomy and Crop Science.

10. Crain, J., Mondal, S., Rutkoski, J., Singh, R. P., & Poland, J. (2018). [Combining high‚Äêthroughput phenotyping and genomic information to increase prediction and selection accuracy in wheat breeding](https://acsess.onlinelibrary.wiley.com/doi/10.3835/plantgenome2017.05.0043). The Plant Genome.   

11. Biswas, A., Andrade, M. H. M. L., Acharya, J. P., de Souza, C. L., Lopez, Y., de Assis, G., ... & Rios, E. F. (2021). [Phenomics-assisted selection for herbage accumulation in alfalfa (Medicago sativa L.)](https://www.frontiersin.org/articles/10.3389/fpls.2021.756768/full). Frontiers in Plant Science.

12. Qu, J., Morota, G., & Cheng, H. (2022). [A Bayesian random regression method using mixture priors for genome‚Äêenabled analysis of time‚Äêseries high‚Äêthroughput phenotyping data](https://doi.org/10.1002/tpg2.20228). The Plant Genome.

13. Wilson, G., Aruliah, D. A., Brown, C. T., Chue Hong, N. P., Davis, M., Guy, R. T., ... & Wilson, P. (2014). [Best practices for scientific computing](https://doi.org/10.1371/journal.pbio.1001745). PLOS Biology.

14. Khaipho-Burch, M., Cooper, M., Crossa, J., de Leon, N., Holland, J., Lewis, R., ... & Buckler, E. S. (2023). [Genetic modification can improve crop yields‚Äîbut stop overselling it](http://dx.doi.org/10.1038/d41586-023-02895-w). Nature.

15. Huang, W., & Mackay, T. F. (2016). [The genetic architecture of quantitative traits cannot be inferred from variance component analysis](https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1006421). PLoS Genetics.

16. Wang, M., Li, R., & Xu, S. (2020). [Deshrinking ridge regression for genome-wide association studies](https://doi.org/10.1093/bioinformatics/btaa345). Bioinformatics, 36(14), 4154-4162.

17. Moreira, F. F., Oliveira, H. R., Volenec, J. J., Rainey, K. M., & Brito, L. F. (2020). [Integrating high-throughput phenotyping and statistical genomic methods to genetically improve longitudinal traits in crops](https://doi.org/10.3389/fpls.2020.00681). Frontiers in Plant Science, 11, 681.

18. Hu, X., Carver, B.F., El-Kassaby, Y.A. et al. (2023) [Weighted kernels improve multi-environment genomic prediction](https://doi.org/10.1038/s41437-022-00582-6). Heredity 130, 82‚Äì91.

19. Alemu, A., √Östrand, J., Montesinos-L√≥pez, O. A., y S√°nchez, J. I., Fern√°ndez-G√≥nzalez, J., Tadesse, W., ... & Chawade, A. (2024). [Genomic selection in plant breeding: key factors shaping two decades of progress](https://doi.org/10.1016/j.molp.2024.03.007). Molecular Plant.

20. Mackay, T. F., & Anholt, R. R. (2024). [Pleiotropy, epistasis and the genetic architecture of quantitative traits](https://www.nature.com/articles/s41576-024-00711-3). Nature Reviews Genetics, 1-19.

21. Saad, N. S. M., Neik, T. X., Thomas, W. J., Amas, J. C., Cantila, A. Y., Craig, R. J., ... & Batley, J. (2022). [Advancing designer crops for climate resilience through an integrated genomics approach](https://doi.org/10.1016/j.pbi.2022.102220). Current Opinion in Plant Biology, 67, 102220.

22. Zhu, H., & Zhou, X. (2020). [Statistical methods for SNP heritability estimation and partition: A review](https://doi.org/10.1016/j.csbj.2020.06.011). Computational and Structural Biotechnology Journal, 18, 1557-1568.

23. Negus, K. L., Li, X., Welch, S. M., & Yu, J. (2024). [The role of artificial intelligence in crop improvement](https://doi.org/10.1016/bs.agron.2023.11.001). Advances in Agronomy, 184, 1-66. 

24. Wang, F., Feldmann, M. J., & Runcie, D. E. (2025). [Why Accuracy Metrics Fall Short in Comparing Phenomic and Genomic Prediction Models](https://doi.org/10.1101/2025.01.09.632209). bioRxiv, 2025-01.

25. Korte, A., & Farlow, A. (2013). [The advantages and limitations of trait analysis with GWAS: a review](https://link.springer.com/article/10.1186/1746-4811-9-29). Plant methods, 9, 1-9.

26. Mumford, M. H., Forknall, C. R., Rodriguez, D., Eyre, J. X., & Kelly, A. M. (2023). [Incorporating environmental covariates to explore genotype√ó environment√ó management (G√ó E√ó M) interactions: A one-stage predictive model](https://www.sciencedirect.com/science/article/pii/S037842902300326X). Field Crops Research, 304, 109133.

27. Hufnagel, B., de Sousa, S. M., Assis, L., Guimaraes, C. T., Leiser, W., Azevedo, G. C., ... & Magalhaes, J. V. (2014). [Duplicate and conquer: multiple homologs of PHOSPHORUS-STARVATION TOLERANCE1 enhance phosphorus acquisition and sorghum performance on low-phosphorus soils](https://academic.oup.com/plphys/article/166/2/659/6113341). Plant physiology, 166(2), 659-677.


