**Aqui est√° uma sele√ß√£o de 12 artigos essenciais que cobrem t√≥picos importantes na √°rea de melhoramento gen√©tico.**

## üå± Sele√ß√£o Gen√¥mica & Gen√©tica Quantitativa

# ‚ù∂ Entendendo o Processo de Sele√ß√£o Gen√¥mica em Plantas  
- https://www.embrapa.br/busca-de-publicacoes/-/publicacao/1155985/melhoramento-de-precisao-aplicacoes-e-perspectivas-na-genetica-de-plantas
- Resende, R.T. Balancing genomic selection efforts for allogamous plant breeding programs. J. Crop Sci. Biotechnol. 27, 349‚Äì358 (2024). https://doi.org/10.1007/s12892-024-00234-7

# ‚ù∑ Como Funciona a Gen√©tica Quantitativa?  
- https://www.linkedin.com/posts/thecrop_breeding-precisionbreeding-agriculture-activity-7125966987658178563-_H7R?utm_source=share&utm_medium=member_desktop

# ‚ù∏ Deriva√ß√µes das Rela√ß√µes Gen√©ticas Entre Indiv√≠duos  
- https://www.linkedin.com/posts/thecrop_breeding-precisionagriculture-genomics-activity-7128515032455245825-8aaz?utm_source=share&utm_medium=member_desktop

## üß¨ Modelos Estat√≠sticos & Teoria

# ‚ùπ Efeitos Fixos vs Aleat√≥rios em Modelos Mistos  
- https://www.linkedin.com/posts/thecrop_mixedmodels-breeding-precisionbreeding-activity-7134863118593081345-osE1?utm_source=share&utm_medium=member_desktop

# ‚ù∫ A Equa√ß√£o do Melhorista Explicada  
- https://www.linkedin.com/posts/thecrop_breeding-phenomics-genomics-activity-7158601205236056064-TzZM?utm_source=share&utm_medium=member_desktop

# ‚ùª Por Que a Herdabilidade √© Fundamental no Melhoramento?  
- https://www.linkedin.com/posts/thecrop_genetics-breeding-heritability-activity-7152750689352327168-m7ld?utm_source=share&utm_medium=member_desktop

## üåæ Popula√ß√µes de Melhoramento & M√©todos

# ‚ùº Entendendo Popula√ß√µes de Melhoramento  
- https://www.linkedin.com/posts/thecrop_thecrop-plantbreeding-genetics-activity-7155311141504577536-p5YJ?utm_source=share&utm_medium=member_desktop

# ‚ùΩ A Complexa Intera√ß√£o Gen√≥tipo √ó Ambiente  
- https://www.linkedin.com/posts/thecrop_gxe-breeding-plantbreeding-activity-7175942157944446976-B_z5?utm_source=share&utm_medium=member_desktop

# ‚ùæ Sele√ß√£o Recorrente ‚Äî √â T√£o Simples Assim?  
- https://www.linkedin.com/posts/thecrop_breeding-genomics-phenomics-activity-7188680080171163648-Cblr?utm_source=share&utm_medium=member_desktop

## üìä ‚Äò√îmicas & Implementa√ß√£o

# ‚ù∂‚ìø Fen√¥mica no Melhoramento Gen√©tico  
- https://www.linkedin.com/posts/thecrop_phenomics-breeding-animalbreeding-activity-7198382693988888576-o0Pu?utm_source=share&utm_medium=member_desktop

# ‚ù∂‚ù∂ A Ascens√£o da Enviromia (Enviromics)  
- https://www.cell.com/molecular-plant/fulltext/S1674-2052(24)00118-7?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS1674205224001187%3Fshowall%3Dtrue

# ‚ù∂‚ù∑ Etapas de Implementa√ß√£o Experimental para o Melhoramento de Culturas  
- https://nph.onlinelibrary.wiley.com/doi/10.1111/nph.19951

**Artigos de pesquisa e de revis√£o voltados √† gen√©tica e melhoramento, gen√©tica quantitativa e gen√¥mica de diversas culturas al√≥gamas e aut√≥gamas.**

1. Moeinizade, S., Kusmec, A., Hu, G., Wang, L., & Schnable, P. S. (2020). [Multi-trait genomic selection methods for crop improvement](https://academic.oup.com/genetics/article/215/4/931/6065544). Genetics.

2. Farooq, M., van Dijk, A. D., Nijveen, H., Mansoor, S., & de Ridder, D. (2023). [Genomic prediction in plants: opportunities for ensemble machine learning based approaches](https://f1000research.com/articles/11-802). F1000Research.

3. Lane, H. M., Murray, S. C., Montesinos‚ÄëL√≥pez, O. A., Montesinos‚ÄëL√≥pez, A., Crossa, J., Rooney, D. K., ... & Morgan, C. L. (2020). [Phenomic selection and prediction of maize grain yield from near‚Äêinfrared reflectance spectroscopy of kernels](https://acsess.onlinelibrary.wiley.com/doi/10.1002/ppj2.20002). The Plant Phenome Journal.

4. Wei√ü, T. M., Zhu, X., Leiser, W. L., Li, D., Liu, W., Schipprack, W., ... & W√ºrschum, T. (2022). [Unraveling the potential of phenomic selection within and among diverse breeding material of maize (Zea mays L.)](https://academic.oup.com/g3journal/article/12/3/jkab445/6509517). G3.

5. Sun, H., Wei, M., Xu, Z., Bai, C., & Sun, B. (2022). [PC‚ÄêDOT: Improving genomic prediction ability of principal component regression by DOT product](https://onlinelibrary.wiley.com/doi/10.1111/age.13255). Animal Genetics.

6. Cheng, J., Maltecca, C., VanRaden, P. M., O'Connell, J. R., Ma, L., & Jiang, J. (2023). [SLEMM: million-scale genomic predictions with window-based SNP weighting](https://academic.oup.com/bioinformatics/article/39/3/btad127/7075542). Bioinformatics.

7. Robert, P., Brault, C., Rincent, R., & Segura, V. (2022). [Phenomic Selection: A New and Efficient Alternative to Genomic Selection](https://link.springer.com/protocol/10.1007/978-1-0716-2205-6_14). In Genomic Prediction of Complex Traits: Methods and Protocols (pp. 397-420). New York, NY: Springer US.

8. Westhues, C. C., Mahone, G. S., da Silva, S., Thorwarth, P., Schmidt, M., Richter, J. C., ... & Beissinger, T. M. (2021). [Prediction of maize phenotypic traits with genomic and environmental predictors using gradient boosting frameworks](https://www.frontiersin.org/articles/10.3389/fpls.2021.699589/full). Frontiers in Plant Science.

9. Piepho, H. P., B√ºchse, A., & Emrich, K. (2003). [A hitchhiker's guide to mixed models for randomized experiments](https://onlinelibrary.wiley.com/doi/abs/10.1046/j.1439-037X.2003.00049.x). Journal of Agronomy and Crop Science.

10. Crain, J., Mondal, S., Rutkoski, J., Singh, R. P., & Poland, J. (2018). [Combining high‚Äêthroughput phenotyping and genomic information to increase prediction and selection accuracy in wheat breeding](https://acsess.onlinelibrary.wiley.com/doi/10.3835/plantgenome2017.05.0043). The Plant Genome.   

11. Biswas, A., Andrade, M. H. M. L., Acharya, J. P., de Souza, C. L., Lopez, Y., de Assis, G., ... & Rios, E. F. (2021). [Phenomics-assisted selection for herbage accumulation in alfalfa (Medicago sativa L.)](https://www.frontiersin.org/articles/10.3389/fpls.2021.756768/full). Frontiers in Plant Science.

12. Qu, J., Morota, G., & Cheng, H. (2022). [A Bayesian random regression method using mixture priors for genome‚Äêenabled analysis of time‚Äêseries high‚Äêthroughput phenotyping data](https://doi.org/10.1002/tpg2.20228). The Plant Genome.

13. Wilson, G., Aruliah, D. A., Brown, C. T., Chue Hong, N. P., Davis, M., Guy, R. T., ... & Wilson, P. (2014). [Best practices for scientific computing](https://doi.org/10.1371/journal.pbio.1001745). PLOS Biology.

14. Khaipho-Burch, M., Cooper, M., Crossa, J., de Leon, N., Holland, J., Lewis, R., ... & Buckler, E. S. (2023). [Genetic modification can improve crop yields‚Äîbut stop overselling it](http://dx.doi.org/10.1038/d41586-023-02895-w). Nature.

15. Huang, W., & Mackay, T. F. (2016). [The genetic architecture of quantitative traits cannot be inferred from variance component analysis](https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1006421). PLoS Genetics.

16. Wang, M., Li, R., & Xu, S. (2020). [Deshrinking ridge regression for genome-wide association studies](https://doi.org/10.1093/bioinformatics/btaa345). Bioinformatics, 36(14), 4154-4162.

17. Moreira, F. F., Oliveira, H. R., Volenec, J. J., Rainey, K. M., & Brito, L. F. (2020). [Integrating high-throughput phenotyping and statistical genomic methods to genetically improve longitudinal traits in crops](https://doi.org/10.3389/fpls.2020.00681). Frontiers in Plant Science, 11, 681.

18. Hu, X., Carver, B.F., El-Kassaby, Y.A. et al. (2023) [Weighted kernels improve multi-environment genomic prediction](https://doi.org/10.1038/s41437-022-00582-6). Heredity 130, 82‚Äì91.

19. Alemu, A., √Östrand, J., Montesinos-L√≥pez, O. A., y S√°nchez, J. I., Fern√°ndez-G√≥nzalez, J., Tadesse, W., ... & Chawade, A. (2024). [Genomic selection in plant breeding: key factors shaping two decades of progress](https://doi.org/10.1016/j.molp.2024.03.007). Molecular Plant.

20. Mackay, T. F., & Anholt, R. R. (2024). [Pleiotropy, epistasis and the genetic architecture of quantitative traits](https://www.nature.com/articles/s41576-024-00711-3). Nature Reviews Genetics, 1-19.

21. Saad, N. S. M., Neik, T. X., Thomas, W. J., Amas, J. C., Cantila, A. Y., Craig, R. J., ... & Batley, J. (2022). [Advancing designer crops for climate resilience through an integrated genomics approach](https://doi.org/10.1016/j.pbi.2022.102220). Current Opinion in Plant Biology, 67, 102220.

22. Zhu, H., & Zhou, X. (2020). [Statistical methods for SNP heritability estimation and partition: A review](https://doi.org/10.1016/j.csbj.2020.06.011). Computational and Structural Biotechnology Journal, 18, 1557-1568.

23. Negus, K. L., Li, X., Welch, S. M., & Yu, J. (2024). [The role of artificial intelligence in crop improvement](https://doi.org/10.1016/bs.agron.2023.11.001). Advances in Agronomy, 184, 1-66. 

24. Wang, F., Feldmann, M. J., & Runcie, D. E. (2025). [Why Accuracy Metrics Fall Short in Comparing Phenomic and Genomic Prediction Models](https://doi.org/10.1101/2025.01.09.632209). bioRxiv, 2025-01.

25. Korte, A., & Farlow, A. (2013). [The advantages and limitations of trait analysis with GWAS: a review](https://link.springer.com/article/10.1186/1746-4811-9-29). Plant methods, 9, 1-9.

26. Mumford, M. H., Forknall, C. R., Rodriguez, D., Eyre, J. X., & Kelly, A. M. (2023). [Incorporating environmental covariates to explore genotype√ó environment√ó management (G√ó E√ó M) interactions: A one-stage predictive model](https://www.sciencedirect.com/science/article/pii/S037842902300326X). Field Crops Research, 304, 109133.

27. Hufnagel, B., de Sousa, S. M., Assis, L., Guimaraes, C. T., Leiser, W., Azevedo, G. C., ... & Magalhaes, J. V. (2014). [Duplicate and conquer: multiple homologs of PHOSPHORUS-STARVATION TOLERANCE1 enhance phosphorus acquisition and sorghum performance on low-phosphorus soils](https://academic.oup.com/plphys/article/166/2/659/6113341). Plant physiology, 166(2), 659-677.
